src/components/BButton/BButton.test.js

```diff
@@ -1,43 +1,43 @@
-import { mount, shallow } from 'vue-test-utils'
-import BButton from './BButton'
+import { mount, shallow } from 'vue-test-utils';
```

Comment:
We don't use `;`

Reply:
Justification from AirBnB style guide: https://github.com/airbnb/javascript#semicolons

My reasoning:
- `eslint --fix` captures it without you having to do anything
- saves some behind the scenes compilation logic
- makes it easier to read for me

Discuss

Reply:
The question I would ask is, "Do semicolons improve the readability of the code?"

I prefer no semicolons.  I can go either way for consistency, but I think it clutters up the code.  JavaScript has automatic semicolon insertion, so semicolons in source really don't do anything.